File: app/src/main/java/com/tomtom/timetoleave/SafeTravelsActivity.java
package com.tomtom.timetoleave;

import android.content.Intent;
import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import androidx.appcompat.widget.Toolbar;
import android.view.View;
import android.widget.Button;

public class SafeTravelsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_safe_travels);

        initToolbarSettings();

        Button btnNextTrip = findViewById(R.id.button_safe_travels_next_trip);
        btnNextTrip.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(SafeTravelsActivity.this, MainActivity.class);
                startActivity(intent);
            }
        });
    }

    private void initToolbarSettings() {
        Toolbar toolbar = findViewById(R.id.custom_toolbar);
        setSupportActionBar(toolbar);

        ActionBar actionBar = getSupportActionBar();
        if (actionBar != null) {
            getSupportActionBar().setHomeAsUpIndicator(R.drawable.ic_close);
            actionBar.setDisplayHomeAsUpEnabled(true);
            actionBar.setDisplayShowHomeEnabled(true);
            actionBar.setDisplayShowTitleEnabled(false);
        }
    }

}

File: TimeToLeave-ktx/src/main/java/com/tomtom/timetoleave/SafeTravelsActivity.kt
package com.tomtom.timetoleave

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar

class SafeTravelsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_safe_travels)

        initToolbarSettings()

        val btnNextTrip = findViewById<Button>(R.id.button_safe_travels_next_trip)
        btnNextTrip.setOnClickListener {
            val intent = Intent(this@SafeTravelsActivity, MainActivity::class.java)
            startActivity(intent)
        }
    }

    private fun initToolbarSettings() {
        val toolbar = findViewById<Toolbar>(R.id.custom_toolbar)
        setSupportActionBar(toolbar)
        supportActionBar?.apply {
            this.setHomeAsUpIndicator(R.drawable.ic_close)
            this.setDisplayHomeAsUpEnabled(true)
            this.setDisplayShowHomeEnabled(true)
            this.setDisplayShowTitleEnabled(false)
        }
    }

}
